app/controllers/view_components_controller.rb

```diff
@@ -59,4 +61,8 @@ def set_locale
   def prepend_application_view_paths
     prepend_view_path Rails.root.join("app/views") if defined?(Rails.root)
   end
+
+  def url_helpers
+    Rails.application.routes.url_helpers
+  end
```

Comment:
Do we need these helpers for anything else? If we don't, what if we just defined `preview_component_path` directly?

```suggestion
  def preview_component_path
    Rails.application.routes.url_helpers.preview_component_path
  end
```

Reply:
Great call! Will update

Reply:
I'm still wondering if we do controllers as part of the engine can we get the URL helpers for free event if we go with this configuration mechanism vs mounting the engine?

Reply:
As long as our controller inherits from Rails::ApplicationController the URL helpers will be needed

Reply:
Hmmm just did some testing and turns out we don't need it, I wonder why I added it in the first place.